lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service.rb

```diff
@@ -11,36 +11,36 @@ class MiqAeService
 
     attr_accessor :logger
 
-    @@id_hash = {}
-    @@current = []
+    @id_hash = {}
+    @current = []
```

Comment:
I have played with a similar change in the past and it ended up not working as expected.  I would prefer we remove this change for now and can maybe look to address in the future.  Thanks.

Reply:
Sure, deleted. I put it in one commit so it is easy to remove 😁